app/views/admin/timelogs/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="well">
+  <% if params[:timelogs] == 'false' %>
+    <h4>This user have no timelogs yet</h4>
+    <%= link_to 'Create timelog', new_admin_user_timelog_path(params[:user_id]), class: 'btn btn-default' %><br>
+  <% else %>
+    <h3>Timelogs</h3>
+    <strong>Filter by:</strong><p><%= link_to "Start time", admin_user_timelogs_path(:filter=>"start_time") %>
+    <%= link_to "Duration", admin_user_timelogs_path(:filter=>"duration") %>
+    <%= link_to "End time", admin_user_timelogs_path(:filter=>"end_time") %>
+    <%= link_to "Ends this week", admin_user_timelogs_path(:filter=>"ends_this_week") %>
+    <%= link_to "Ends this month", admin_user_timelogs_path(:filter=>"ends_this_month") %></p>
+    <p><strong>Or select date range</strong></p>
+    <form id="dateForm">
+      <%= form_tag(admin_user_timelogs_path(params[:user_id]), :method => "get") do %>
+        <p><%= date_field_tag :start_date %></p>
+        <p><%= date_field_tag :end_date %></p>
+        <p><%= submit_tag "Apply", :id => "applyDateButton", :style => 'visibility: hidden' %></p>
+      <% end %>
+    </form>
+    <%= link_to "Download PDF", { controller: "admin/timelogs", action: "download_pdf", user_id: params[:user_id], filter: params[:filter], start_date: params[:start_date], end_date: params[:end_date] }, class: 'btn btn-default', target: "_blank", method: :post %>
```

Comment:
It is better to user `path_helpers`, not to set `controller` and `action` separately.